packages/cli/src/commands/node/install.ts

```diff
@@ -0,0 +1,52 @@
+import { Command, Flags } from "@oclif/core";
+import { ensureSwankyProject, getSwankyConfig, Spinner, downloadNode, swankyNode } from "@astar-network/swanky-core";
+import inquirer = require("inquirer");
+import path = require("node:path");
+import { writeJSON } from "fs-extra";
+import { choice } from "../../lib/prompts";
+
+export class InstallNode extends Command {
+  static description = "Install swanky node binary";
+
+  static flags = {
+    verbose: Flags.boolean({ char: "v" }),
+  };
+
+  async run(): Promise<void> {
+    await ensureSwankyProject();
+
+    const { flags } = await this.parse(InstallNode);
+    const config = await getSwankyConfig();
+    const spinner = new Spinner(flags.verbose);
+
+    const projectPath = path.resolve();
+
+    if (config.node.localPath !== "") {
+        const answers = await inquirer.prompt([
```

Comment:
Let's try to gradually phase out inquirer.
This seems like a great use case for https://github.com/oclif/core/blob/main/src/cli-ux/README.md